app/src/main/java/org/oppia/app/walkthrough/WalkthroughActivity.kt

```diff
@@ -21,4 +23,13 @@ class WalkthroughActivity : InjectableAppCompatActivity(), WalkthroughFragmentCh
   override fun onBackPressed() {
     walkthroughActivityPresenter.handleSystemBack()
   }
+  companion object {
+    internal const val WALKTHROUGH_ACTIVITY_PROFILE_ID_KEY = "WalkthroughActivity.internal_profile_id"
```

Comment:
Rename key:
`WALKTHROUGH_ACTIVITY_PROFILE_ID_KEY ` to `WALKTHROUGH_ACTIVITY_INTERNAL_PROFILE_ID_KEY `